### ISSUE TITLE
Possible to add features

### ISSUE THREAD

**Issue by eduardomart** [cxhawk/liveSubs#3]
Sometimes in a conversation being able to have line breaks is necessary for example:

-are you there?
-yes I am

also (but less important) italics, is there a way to get these working? thanks

**Comment by cxhawk** [comment]
yes, it's possible to add this feature, also useful for multilingual subs

I'm glad people start using this now
